web/src/components/vis/AnovaTable.vue

```diff
@@ -0,0 +1,64 @@
+<script>
+import { format } from 'd3-format';
+
+export default {
+
+  props: {
+    data: {
+      validator: prop => !prop || ('data' in prop && 'pairs' in prop),
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      format: format('.2e'),
+    };
+  },
+
+  computed: {
+    items() {
+      return (this.data && this.data.data) || [];
+    },
+    pairs() {
+      return (this.data && this.data.pairs) || [];
+    },
+    headers() {
+      return [
+        {
+          text: 'Metabolite',
```

Comment:
This is probably fine, but we may want to consider keeping these components more general purpose, then specializing them to this application. We may also *not* want to consider that, since we *are* developing a metabolomics application here.